Byron's submission/js/config.js

```diff
@@ -0,0 +1,2327 @@
+Contents:
+*/
+
+/*
+########################################################################
+HEADER SECTION
+*/
+let topTitleDiv = "<h4>2024 MUSA 611 JavaScript for Planners Story Map Project</h4>";
+let titleDiv = "<h1>How accessible is your new home?</h1>";
+let bylineDiv = "<p>Byron Huang<br>18 Sep 2024</p>";
+let descriptionDiv = `
+<p>Welcome!</p>
+<p>In October 2024, the government of Singapore will be launching <span style="background-color: yellow;">15 new public housing projects</span> for <a href="https://homes.hdb.gov.sg/home/landing" target="_blank">sale</a>. If you are a prospective homebuyer, how would you choose which project to apply to?</p>
+<p>This story map is aimed at illustrating the accessibility of these upcoming public housing projects. One common pain point for prospective homebuyers is comparing the accessibility of different projects.</p>
+<p>To aid in the comparison process, the government releases Town Maps (such as the one pictured below), but these are static and comparing projects within the same launch is still not straightforward.</p>
+<p> As such, this story map addresses this issue by experimenting with new ways of comparing sites within the same sales launch, using various accessibility indicators.</p>
+<p><br></p>
+<div style="max-width:70%; text-align:center; margin-left:auto; margin-right:auto">
+  <img src="./data/images/sample_townmap.webp" alt="Sample town map" style="max-width:75%; height:auto;">
+</div>
+<p><em>Above: An example of a static Town Map provided by the Singapore government for prospective buyers, showing parks, cycle paths, stations, and some neighborhood amenities.</em></p>
+<p><br></p>
+<p>To facilitate comparison between different sites, accessibility metrics are calculated in ArcGIS and presented in the map views below. This story map examines the following indicators:</p>
+<p><strong>1. Accessibility to nature</strong><br>Area (in km<sup>2</sup>) of parks and nature reserves within a 1km buffer from site</p>
+<p><strong>2. Accessibility to transport</strong><br>
+Distance (in m) to nearest MRT/LRT station<br>
+Length (in m) of cycle paths within a 1km buffer from site</p>
+<p><strong>3. Accessibility to existing neighbourhood amenities</strong><br>
+Distance (in m) to nearest hawker centre<br>
+Distance (in m) to nearest library<br>
+Area (in km<sup>2</sup>) of existing public housing within a 1km buffer from site</p>
+<p><br></p>
+<p style="text-align:center">Explore the projects below<br>▼</p>
+`;
+
+/*
+########################################################################
+CHAPTERS
+*/
+
+let divChapter1 =`
+<h3>Existing public housing and the Built-to-Order (BTO) system</h3>
+<p>The share of public housing in Singapore's housing market is particularly high in the world. This map shows the extent of public housing in Singapore (in <span style="color:#b20300;">red</span>). <strong>About 80% of Singaporeans live in public housing.</strong></p>
```

Comment:
**praise:**I like the practice of changing the text/background color when mentioning the color.